scripts/release_scripts/deploy.py

```diff
@@ -465,8 +465,8 @@ def execute_deployment():
         raise Exception(
             'The deployment script must be run from a release or test branch.')
     if common.is_current_branch_a_test_branch() and (
-            app_name == APP_NAME_OPPIASERVER):
-        raise Exception('Test branch cannot be deployed to prod.')
+            app_name in [APP_NAME_OPPIASERVER, APP_NAME_OPPIATESTSERVER]):
```

Comment:
DO we have any APP_NAME_* for the backup server as well?

Reply:
No, we don't.